tests/test_0965-inverted-axes-variances-hist-888.py

```diff
@@ -2,10 +2,10 @@
 
 import pytest
 import numpy
-import hist
 import uproot
 import skhep_testdata
 
+hist = pytest.importorskip("hist")
```

Comment:
`hist` is not used in these tests, thus it can be simply removed as an import

Reply:
```suggestion
```